setup.py

```diff
@@ -244,8 +244,13 @@ def get_packages():
       platforms=['any'],
       packages=get_packages(),
       package_data={LIBNAME: get_package_data(LIBNAME, EXTLIST),
-                    'spyderplugins':
-                    get_package_data('spyderplugins', EXTLIST)},
+                    'spyplugins':
+                    get_package_data('spyplugins', EXTLIST+['.py']),
```

Comment:
Another thing: doesn't `get_package_data` works recursively? It would seem like that given that before it was only necessary to pass `spyderplugins` to it :-)